src/main/java/org/opensearch/sdk/NettyTransport.java

```diff
@@ -85,22 +99,18 @@ public Netty4Transport getNetty4Transport(Settings settings, ThreadPool threadPo
      * Initializes the TransportService object for this extension. This object will control communication between the extension and OpenSearch.
      *
      * @param settings  The transport settings to configure.
-     * @param extensionsRunner method to call
      * @return The initialized TransportService object.
      */
-    public TransportService initializeExtensionTransportService(Settings settings, ExtensionsRunner extensionsRunner) {
+    public TransportService initializeExtensionTransportService(Settings settings) {
 
         ThreadPool threadPool = new ThreadPool(settings);
 
         Netty4Transport transport = getNetty4Transport(settings, threadPool);
 
-        // Stop any existing transport service
-        if (extensionsRunner.extensionTransportService != null) {
-            extensionsRunner.extensionTransportService.stop();
-        }
+        // delete stop
```

Comment:
This looks like a note-to-self to remove, so you can probably remove it.   

Note for other reviewers: I added the stop/start code myself at some point thinking it was needed, but it's not.